### PR TITLE
Fix the issue with the pet attribute sliders not having a default val…

### PR DESCRIPTION
**Pull request by zehash**
…ue stored in them.

https://www.pivotaltracker.com/n/projects/1108894/stories/102316738

Added default values in a new migration and set them from there. Also modified the pet model.
#102316738

### CODE REVIEW

File: app/models/pet.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Pet < ActiveRecord::Base
   validates_presence_of :name, :pet_type_id, :size_id, :pet_age, :sex_id
   validates_presence_of :other_pet_type, if: proc {|pet| pet.pet_type_id == 5} # when pet type is 'other'
   validates_inclusion_of :pet_type_id, :in => ReferenceData::PetType.all.map(&:id)
-  validates :energy_level, inclusion: { in: 1..5 } # Low, low medium, medium, high medium, and high
+  validates :energy_level, inclusion: { in: %w(1 2 3 4 5) } # Low, low medium, medium, high medium, and high
```
Comment (zero51) [inline app/models/pet.rb]: So energy level should be a string, whereas size and sex IDs are integers?

File: db/migrate/20151004053415_set_default_value_to_pet.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class SetDefaultValueToPet < ActiveRecord::Migration
+  def change
+    change_column :pets, :energy_level, :string,  :limit => 255, :default => "3"
```
Comment (zero51) [inline db/migrate/20151004053415_set_default_value_to_pet.rb]: You're changing the energy level type?

File: app/models/pet.rb (hunk 1 above)
Comment (zehash) [inline app/models/pet.rb]: We are using reference data for size and sex. If you look at how we map them using reference data, then yes, I believe they should be.

File: db/migrate/20151004053415_set_default_value_to_pet.rb (hunk 2 above)
Comment (zehash) [inline db/migrate/20151004053415_set_default_value_to_pet.rb]: I didnt change datatype
I just added default values

size was only integer rest two were strings

Comment (zero51) [inline db/migrate/20151004053415_set_default_value_to_pet.rb]: Please look at the changes to schema.rb below, or at your local DB, or in the console - `energy_level` has changed from integer to string.

```
irb(main):004:0> Pet
=> Pet(id: integer, user_id: integer, name: string, breed: string, created_at: datetime, updated_at: datetime, dislike_loneliness: boolean, dislike_children: boolean, dislike_animals: boolean, dislike_people: boolean, flea_treated: boolean, vaccinated: boolean, house_trained: boolean, age: string, microchip_number: string, council_number: string, explain_dislikes: text, other_pet_type: string, emergency_contact_name: string, emergency_contact_phone: string, vet_name: string, vet_phone: string, medication: text, date_of_birth: date, pet_type_id: integer, sex_id: integer, size_id: integer, energy_level: integer, personalities: text)
```

Comment (zehash) [inline db/migrate/20151004053415_set_default_value_to_pet.rb]: Oh damn - I did change that data type. Accident. I just wanted to add default values. Don't merge that. Ill making the changes and sending them in. Good eye.

**Comment by zero51** [comment]
I think you've made a mistake with the PR. You want to merge it into master?

**Comment by zero51** [comment]
Please remember to use labels to identify PR status. If you want it reviewed, then set the label to 'needs review' and the assignee to me.

**Comment by zehash** [comment]
Yea I saw that, but the reason I want too merge into develop was because today we do the weekly push where we merge develop into master

**Comment by zero51** [comment]
Please be careful with your commit messages. Review the style guide for best practice (see https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message).

**Comment by zero51** [comment]
@zehash You will **always** want to merge into develop (unless it's a hotfix). Deployments to production are handled by myself via releases.

**Comment by zehash** [comment]
Makes sense. What do you want me to do then. Resubmit this as a hotfix?

**Comment by zero51** [comment]
No, definitely not. Remember that hotfixes should only be reserved for critical bug fixes. They will be flagged as such in Pivotal. The PR needs to be corrected so that the target is develop.

**Comment by zehash** [comment]
So I should change the ticket to feature instead of bug right?

**Comment by zero51** [comment]
No, it's still a bug. The Pivotal story type is correct. This PR is incorrect.

**Pull request closed without merging**